docs/reference/billing/request.md

```diff
@@ -136,11 +136,7 @@ Which URL to redirect the merchant to after the charge is confirmed.
 
 Whether to return the `confirmationUrl` as a `string`, or to return a more detailed object (see below).
 
-### trialDays
-
-`number`
-
-Override value for the `trialDays` config option. Only applies to recurring purchases.
+> **Note**: You can override any billing configuration fields at request time by passing them into your `billing.request` call.
```

Comment:
Lets add an example here!

```suggestion
### Plan Overrides
You can override any billing configuration fields at request time by passing them into your `billing.request` call.

```
  const billingResponse = await shopify.billing.request({
    session: callback.session,
    plan: 'My billing plan',
    isTest: true,
    trialDays: 4, // Overrides the trial days set in 'My billing plan'
    amount: 13 // Overrides the amount set in 'My billing plan'
    
    returnObject: true,
  });

  res.redirect(billingResponse.confirmationUrl);
});
```
```